### PR TITLE
Barl

### PR DESCRIPTION
**Pull request by Balvine**

### CODE REVIEW

**Comment by kelvinndmo** [comment]
Fix the merge conflicts

**Comment by kelvinndmo** [comment]
@Balvine What exactly does this pull request do?

**Comment by kelvinndmo** [comment]
Rename the title and add the description

File: home/urls.py [hunk 1]
```diff
@@ -6,8 +6,12 @@
 
 urlpatterns = [
 
-        url(r'^review/(?P<user_id>\d+)', views.Review.as_view()),
-        # url(r'^api/get-review/(?P<pk>[0-9]+)/', views.review.as_view()),
+        # url(r'^review/(?P<user_id>\d+)', views.Review.as_view()),
+        # url(r'^api/review/(?P<pk>[0-9]+)/', views.Review.as_view()),
+        url(r'^api/review/$', views.RevList.as_view()),
```
Comment (kelvinndmo) [inline home/urls.py:11]: Avoid using Regex in URLs, use path from Django 2+

`path('api/review', views.RevList.as_view())`

**Review by kelvinndmo — CHANGES_REQUESTED** [review]

File: home/urls.py (hunk 1 above)
Comment (Balvine) [reply home/urls.py:11]: i change it

**Pull request merged**